pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -240,6 +240,36 @@ func (b *OFBridge) DeleteMeterAll() error {
 	return nil
 }
 
+func (b *OFBridge) GetMeterStats(handleMeterStatsReply func(packetCounts map[int]int64)) error {
+	const OFPM_ALL = 0xffffffff // Represents all meters
+	mpMeterStatsReq := openflow15.NewMpRequest(openflow15.MultipartType_MeterStats)
+	meterMPReq := openflow15.NewMeterMultipartRequest(OFPM_ALL)
+	mpMeterStatsReq.Body = append(mpMeterStatsReq.Body, meterMPReq)
+	ch := make(chan *openflow15.MultipartReply, 1)
+	b.multipartReplyChs[mpMeterStatsReq.Xid] = ch
```

Comment:
I think the bug existed before your PR, but it seems to me that the `b.multipartReplyChs` map is accessed concurrently by multiple goroutines, and we should protect it with a mutex?

cc @wenyingd